### PR TITLE
PXB-3371 Update the release notes for 8.0.35-31 with ARM64.DEB inform…

### PR DESCRIPTION
**Pull request by patrickbirch**
…ation

	modified:   docs/release-notes.md
	new file:   docs/release-notes/8.0/8.0.35-31.0.upd.md
	modified:   mkdocs-base.yml

### CODE REVIEW

**Review by satya-bodapati — APPROVED** [review]
LGTM!

File: variables.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # variables for HTML output
 
-release: '8.0.35-31'
+release: '8.0.35-31
```
Comment (alina-derkach-oaza) [inline variables.yml]: Close the variable with '

File: docs/release-notes/8.0/8.0.35-31.0.upd.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Percona XtraBackup 8.0.35-31-Update (2024-09-20)
+
+This update to the release of Percona XtraBackup includes the addition of `arm64.deb` packages in Percona Software Downloads for the following platforms:
```
Comment (alina-derkach-oaza) [inline docs/release-notes/8.0/8.0.35-31.0.upd.md]: Percona Software Downloads - add the link

File: docs/release-notes/8.0/8.0.35-31.0.upd.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+* Debian 11
+
+* Ubuntu 24.04
+
+* Ubuntu 22.04
+
+* Ubuntu 20.04
+
+This extension means Percona XtraBackup is now available for users on ARM-based systems.
```
Comment (alina-derkach-oaza) [inline docs/release-notes/8.0/8.0.35-31.0.upd.md]: It has been available before too but for other platforms.

File: docs/release-notes.md [hunk 4]
```diff
@@ -1,6 +1,8 @@
 # Percona XtraBackup 8.0 release notes index
 
-* [Percona XtraBackup 8.0.35-31 (2024-06-20)](release-notes/8.0/8.0.35-31.0.md)
+* [Percona XtraBackup 8.0.35-31 Update (2024-09-20)](release-notes/8.0/8.0.35-31.0.upd.md)
+
+* [Percona XtraBackup 8.0.35-31 (2024-06-20)](release-notes/8.0/8.0.35-31.0.upd.md)
```
Comment (alina-derkach-oaza) [inline docs/release-notes.md]: The path is incorrect. It should be release-notes/8.0/8.0.35-31.0.md

**Review by alina-derkach-oaza — APPROVED** [review]
LGTM

**Pull request merged**